test-network-function/lifecycle/suite.go

```diff
@@ -313,7 +326,7 @@ func testPodsRecreation(env *config.TestEnvironment) {
 	})
 }
 
-func closeOcSessions(containers map[configsections.ContainerIdentifier]*config.Container, nodeName string) {
+func closeOcSessionsByNode(containers map[configsections.ContainerIdentifier]*config.Container, nodeName string) {
```

Comment:
it's better to place ```closeOcSessionsByNode``` and ```closeOcSessionsByDeployment``` since they are doing almost the same thing

Reply:
updated